zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -162,6 +165,17 @@ public void export(final Record<?> record) {
     }
 
     writer.addRecord(record);
+
```

Comment:
❌ There is one issue with this. We can't cache forever and everything. We should have a limit of how much we put into our writer. If we reached the size we should fail, to loop in the exporter director.